### DISCUSSION TITLE
packaging on conda-forge

### DISCUSSION THREAD

**Discussion by brendanf** [wlandau/crew#80] [General]
@wlandau and  @shikokuchuo , I'd like to package `crew` and its dependencies (for this context, `nanonext` and `mirai`) for conda-forge.  Do you want me to include you as maintainers of the conda-forge recipes for your respective packages (a distinct role from maintainer of the actual R package on CRAN)? In practice this would mean getting a few automated emails each time conda-forge finds an updated version on CRAN. The most common result is that it successfully updates the conda package automatically and there is no need for human intervention; in some cases (particularly where package dependencies have changed in a non-trivial way) there may need to be a bit more done.  I and the general R package recipe group would be maintainers in either case, so it's totally up to you.

**Comment by wlandau** [comment]
I am fine either way. If you think the role adds extra assurance for the conda-forge recipes, then sure. Otherwise, I am fine to be omitted.

**Comment by shikokuchuo** [comment]
Likewise, as you prefer. I don't mind a few automated emails if it's helpful for you. But if purely out of courtesy, I am happy for you to be the maintainer for conda-forge.
